Update Blog tutorial

Tosch110 opened the pull request:
Closes #447 

Updates Starport to v0.13.1

fadeev commented:
@Tosch110 great work! 🔥

fadeev commented on blog/tutorial/01-index.md:
````diff
@@ -186,15 +194,15 @@ func (msg MsgCreatePost) ValidateBasic() error {
-Begin by importing your new blog types that we created:
+Begin by checking if the new blog types that we created is imported:
 
 ```go
````
```go
import (
  // Existing imports...
  "github.com/example/blog/x/blog/types"
)
```

@Tosch110 can you please double-check, but I believe the default scaffold already imports `types` package, so if a user includes this import (and their editor isn't smart), Starport throws an error.

Tosch110 replied:
@fadeev this is correct, that's why I changed the wording above the sentence. Do you think we should rather erase this part then?

fadeev replied:
You're right, my bad 🙂

fadeev commented:
I'm getting an error with `x/blog/keeper/querier.go`. It seems like we're importing `"github.com/cosmos/cosmos-sdk/codec"`, but not using `codec` anywhere.

fadeev commented:
Thanks for the quick changes! I believe this is the last thing I have a problem with. In the "List posts with HTTP" section in `x/blog/client/rest/rest.go` imports are unused:

```
import (
  // Existing imports...
  "fmt"
  "net/http"
  "github.com/cosmos/cosmos-sdk/types/rest"
)
```

The UI also has changed, so in web UI section the only change we need is to make:

```
    <div class="sp-container">
      <sp-sign-in />
      <sp-bank-balances />
      <sp-token-send />
      <!-- this line is used by starport scaffolding # 4 -->
      <sp-type-form type="posts" :fields="['title', 'body']"/>
    </div>
```

We can add the following image as the illustration:

<img width="1373" alt="Screenshot 2021-01-08 at 00 34 30" src="https://user-images.githubusercontent.com/332151/103936500-50809a80-5149-11eb-8a17-893f26fcc086.png">

We may also need to remove the section on how to do it manually, since it's using `@tendermint/sig`, and it's no longer maintained.

Tosch110 commented:
Updated as requested, removed the manual `frontend` explanation and code from the repo.

I've updated the `sdk-tutorials/blog/blog` deployment from previous versions to boostrapped with Starport v0.13.1

fadeev reviewed (approved):
Looks good to me! 👍
